invoice/models.py

```diff
@@ -176,10 +177,21 @@ def total(self) -> Decimal:
     @property
     def list_export(self):
         """Get the fields as list with formatted fields of totals."""
-        return [self.name, self.description, self.quantity, self.price, self.tax_string,
-                f'{self.net_total:.2f}', f'{self.total:.2f}']
+        return [self.name,
+                self.description,
+                self.quantity_string,
+                f'{self.price:.2f}',
+                self.tax_string,
+                f'{self.net_total:.2f}',
+                f'{self.total:.2f}']
+
+    @property
+    def quantity_string(self) -> str:
+        """Get the quantity string."""
+        return f'{self.quantity:.2f}'.rstrip('0').rstrip('.,')
```

Comment:
Why do you allow above 4 decimals places but here only 2?

Reply:
quantity had 2 decimal places, so we cut to two here.
taxes has 4, but is multiplied by 100, so two should cover all significant digits